Presentation.Web/Models/API/V2/Internal/Request/Organizations/OrganizationGridPermissionsRequestDTO.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Presentation.Web.Models.API.V2.Internal.Request.Organizations
+{
+    public class OrganizationGridPermissionsRequestDTO
+    {
+        public Guid OrgUuid { get; set; }
```

Comment:
What's this request dto used for?

Reply:
Nvm, i see that it's not needed, since the orgUuid is part of the url